src/clusterfuzz/_internal/bot/tasks/task_creation.py

```diff
@@ -248,7 +248,9 @@ def create_tasks(testcase):
     testcase.put()
     return
 
-  if environment.is_minimization_supported():
+  if environment.is_minimization_supported() or environment.get_value(
+    'MINMIZE_FUZZER_OVERRIDE'
```

Comment:
misspelling? 

Also, would it make sense to just move this env var check into `is_minimization_supported` ?

Reply:
I think it makes sense not to treat minimization as supported here: https://github.com/google/clusterfuzz/pull/3787/files#diff-e8255271eeeadc2bda46b215fbc7b0bb160ef1116f6e27ff895990d88caafa5fR382